src/builder.py

```diff
@@ -130,12 +133,12 @@ def host(self):
     def rename(self, dst):
         os.rename(self._path, dst._path)
 
-    def cmake(self, args=[]):
+    def cmake(self, args: List[Any] = []) -> 'FileHandle':
```

Comment:
This should be `List[str]`, right?

Reply:
yes, I suppose so.

monkeytype didn't figure that out and `List[Any]` didn't cause type checking to fail, so I didn't notice it.